CreateNotification/handler.ts

```diff
@@ -115,207 +114,208 @@ export type CreateNotificationResult = t.TypeOf<
 /**
  * Returns a function for handling createNotification
  */
-// eslint-disable-next-line max-lines-per-function
+// eslint-disable-next-line max-lines-per-function,@typescript-eslint/explicit-function-return-type
 export const getCreateNotificationHandler = (
   lNotificationModel: NotificationModel,
   lDefaultWebhookUrl: HttpsUrl,
   lSandboxFiscalCode: FiscalCode,
   lEmailNotificationServiceBlackList: ReadonlyArray<ServiceId>,
   lWebhookNotificationServiceBlackList: ReadonlyArray<ServiceId>
-) => async (context: Context, input: unknown): Promise<unknown> => {
-  const inputOrError = CreateNotificationInput.decode(input);
-  if (E.isLeft(inputOrError)) {
-    context.log.error(
-      `${context.executionContext.functionName}|Unable to parse getCreateNotificationHandlerInput`
+) =>
+  withJsonInput(async (context, input) => {
```

Comment:
This line is the actual change: handler function has been wrapped by `withJsonInput` decorator